packages/webapp/src/polyfillDateTimeFormat.ts

```diff
@@ -0,0 +1,12 @@
+// We are polyfilling because Chrome desktop has incomplete support for some of the languages (even if in theory they are supported)
+// Once Chrome starts returning the right thing these imports (and the package) should be removed
+import '@formatjs/intl-datetimeformat/polyfill-force';
+import '@formatjs/intl-datetimeformat/add-all-tz';
+import '@formatjs/intl-datetimeformat/locale-data/en';
+import '@formatjs/intl-datetimeformat/locale-data/es';
```

Comment:
If there is any way to future proof this using a vite glob import or something that could be useful to not have to manually update in the future. Not a blocker of course!

Reply:
I'm not sure, since in this case we don't want to import all the locale data but only the languages we support, so glob probably wouldn't work? (I haven't used it before so I might be wrong!). We could maybe do a dynamic import with `supportedLanguages`